src/VisualStudio/Core/Def/Implementation/Library/FindResults/LibraryManager_GoToDefinition.cs

```diff
@@ -107,7 +107,6 @@ private AbstractTreeItem CreateTreeItem(INavigableItem item, int commonPathEleme
             {
                 var childItems = CreateNavigableItemTreeItems(item.ChildItems, commonPathElements);
                 result.Children.AddRange(childItems);
-                result.SetReferenceCount(childItems.Count);
```

Comment:
This never had any effect.  'result' is of type 'SourceReferenceTreeItem'.  And that type never overrode SetReferenceCount to do anything.